shiksha-migration/cohort-members-userdb-migration.js

```
@@ -0,0 +1,194 @@
+      GROUP BY cm."userId", cm."cohortId", cay."cohortAcademicYearId"
+    `;
+
+    const membersRes = await destClient.query(
+      membersQuery,
+      [tenantId, academicYearId]
+    );
+
+    const members = membersRes.rows;
+
+    console.log(
+      `[COHORT_MEMBERS_ACADEMIC_YEAR] Found ${members.length} active Lead/Instructor members from previous year`
```

ESLint

[error] Insert `,`

(prettier/prettier)